app/controllers/reports/regions_controller.rb

```diff
@@ -117,7 +117,7 @@ def monthly_district_data_report
         raise ActiveRecord::RecordNotFound, "unknown report_scope #{report_scope}"
       end
     }
-    @period = Period.month(Date.current)
+    @period = Period.month(params[:period_end] || Date.current)
```

Comment:
The reason I named it something else is that it actually represents something different. Elsewhere in the downloads, it's used to define the type of period not the value: `period: :month`. In this case, it's being used to pass along the period value. I was trying to disambiguate it, but I'll agree that period_end isn't a very good name. `report_date` or something similar might have been better. Regardless, I'll change it back to period because I don't think it should have much impact.